src/main/webapp/css/styles.css

```diff
@@ -291,15 +291,18 @@ button, .file-button {
   -webkit-transition: all 1s;
```

Comment:
The style changes need to go into the sass file and then be compiled, I can do that in the next update to the styles that I do